node/node.go

```diff
@@ -667,9 +664,16 @@ func (node *Node) Start() error {
 						context.TODO(), openBox,
 					)
 					if err != nil {
-						// TODO (lc): block peers sending error messages
-						errChan <- withError{err, msg.GetFrom()}
-						return libp2p_pubsub.ValidationReject
+						switch err {
+						case errIgnoreBeaconMsg:
+							// ignore the further processing of the ignored messages as it is not intended for this node
+							// but propogate the messages to other nodes
+							return libp2p_pubsub.ValidationAccept
```

Comment:
should you add the result to "ValidatorData" before return?

Reply:
No need. The ValidatorData will be used by the real node message handler.  Since we are not processing those messages by the real handler, but just forward the message in p2p layer.  We don't need to add the result to validatorData.  Those ignored messages won't be sent to node message handlers.  It is similar to the ignored consensus message, in https://github.com/harmony-one/harmony/blob/d829d227dc843bb257a5865a1c988332b24efdcf/node/node.go#L649